CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 # General cmake settings
 ###
 
-cmake_minimum_required(VERSION 3.10.2)
+cmake_minimum_required(VERSION 3.7.2)
```

Comment:
Please restore the previous version. This has just been altered previously along with updated os-support.

Reply:
Fixed in commit d98d3a50edbbf984b1fe842908e1420d26c3ff0f.